projectEuler3.py

```diff
@@ -0,0 +1,26 @@
+#Problem 3
+#The prime factors of 13195 are 5, 7, 13 and 29.
+#What is the largest prime factor of the number 600851475143 ?
+def main():
+    number = int(raw_input('Please Enter an Integer: '))
```

Comment:
this is actually above my expectation, good job.

but for the later two, you could simply hard code the input, will be easier that way, esp for the next one.